app/admin_model.py

```diff
@@ -753,12 +753,37 @@ def index(self):
             if not alias and not user and not mailbox:
                 no_match = True
 
+        def user_mailboxes(user_id: int) -> list[Mailbox]:
+            return (
+                Mailbox.filter_by(user_id=user_id)
+                .order_by(Mailbox.id.asc())
+                .limit(10)
+                .all()
+            )
+
+        def user_mailboxes_count(user_id: int) -> int:
+            return Mailbox.filter_by(user_id=user_id).order_by(Mailbox.id.asc()).count()
+
+        def user_aliases(user_id: int) -> list[Alias]:
+            return (
+                Alias.filter_by(user_id=user_id)
+                .order_by(Alias.id.asc())
+                .limit(10)
```

Comment:
this means we only return 10 aliases right?

Reply:
Yep, if there are more than 10 in the template we explicitly mention we only show the first 10.

Reply:
This is to solve some timeouts when checking account with loads of aliases.